go.mod

```diff
@@ -70,14 +70,17 @@ require (
 	github.com/tklauser/go-sysconf v0.3.5 // indirect
 	github.com/tyler-smith/go-bip39 v1.0.1-0.20181017060643-dbb3b84ba2ef
 	github.com/xsleonard/go-merkle v1.1.0
+	go.opentelemetry.io/otel v1.2.0
+	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.2.0
+	go.opentelemetry.io/otel/sdk v1.2.0
 	golang.org/x/crypto v0.0.0-20210322153248-0c34fe9e7dc2
 	golang.org/x/net v0.0.0-20210805182204-aaa1db679c0d // indirect
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
 	golang.org/x/sys v0.0.0-20210816183151-1e6c022a8912
 	golang.org/x/text v0.3.6
 	golang.org/x/time v0.0.0-20210220033141-f8bda1e9f3ba
-	google.golang.org/grpc v1.37.0
-	google.golang.org/protobuf v1.26.0-rc.1
+	google.golang.org/grpc v1.42.0
```

Comment:
Did you run ```go mod tidy```? I do not like when protobuf and grpc update. Could this be a forced update from openTelemetry? @victorcoder

Reply:
Yes. But, they got forced updated on importing the `go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc` package.